fe/fe-core/src/main/java/org/apache/doris/analysis/ShowColumnStatsStmt.java

```diff
@@ -67,4 +70,9 @@ public ShowResultSetMetaData getMetaData() {
         }
         return builder.build();
     }
+
+    public List<String> getPartitionNames() {
```

Comment:
I suggest that if the statistics setting function of partition table data has not been implemented at all, it is best to write todo directly, and the function `getPartitionNames` is not needed.
Then you can directly add the syntax, constructor and get method together.

Reply:
In this PR, the relevant methods have been implemented, but the corresponding syntax has not been added(to be added together in the future, and users can not use it now). this TODO is added to avoid compilation failure.